pkgs/development/python-modules/oslo-db/default.nix

```diff
@@ -51,13 +51,15 @@ buildPythonPackage rec {
     oslo-context
     oslotest
     stestr
-    psycopg2
+    psycopg2-binary
```

Comment:
We don't want to use the binary variant but patch instead it to the normal variant.